server/index.js

```diff
@@ -12,7 +14,14 @@ app.use(bodyParser.urlencoded({ extended: false }));
 
 // respond with "hello world" when a GET request is made to the homepage
```

Comment:
probably would be good to change this comment
also, are you just sending back data.songs as a placeholder to assure it can read the data?

Reply:
Yeah, I was testing it through postman and seeing if I had the correct references (I initially did not).